examples/fabric/src/main/java/dev/rollczi/example/fabric/ExampleFabric.java

```diff
@@ -0,0 +1,19 @@
+package dev.rollczi.example.fabric;
+
+import dev.rollczi.example.fabric.command.BanCommand;
+import dev.rollczi.litecommands.fabric.LiteFabricFactory;
+import net.fabricmc.api.ModInitializer;
+
+/**
+ * 2024/2/13<br>
+ * LiteCommands<br>
+ *
+ * @author huanmeng_qwq
```

Comment:
Generally, I prefer to leave it without the information that it is LiteCommands.
```suggestion
/**
 * @author huanmeng_qwq
```
Also I'm not sure if I really want to leave authors in the comments. If you want, we can add a "Platforms" section in README.md with a signature who was the author of a specific platform implementation. What do you think about that?

Reply:
My IDEA configuration is automatically generated, it seems that your suggestion is very good